src/main/java/com/project/attendancemanagementsystem/controller/AdminController.java

```diff
@@ -0,0 +1,39 @@
+package com.project.attendancemanagementsystem.controller;
+
+import com.project.attendancemanagementsystem.model.StudentRequest;
+import com.project.attendancemanagementsystem.model.StudentResponse;
+import com.project.attendancemanagementsystem.service.AdminService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
```

Comment:
Please use `@RequestMapping("/admin")` annotation
This way you add "/admin" path to all methods in this controller

Reply:
done

Reply:
You should have added `@RequestMapping("/admin")` to `AdminController`

Reply:
done